roles/confluent.variables/defaults/main.yml

```diff
@@ -26,7 +26,7 @@ fips_jar_path: /usr/share/java/kafka/bc-fips-*.jar
 fips_provider_class: org.bouncycastle.jcajce.provider.BouncyCastleFipsProvider
 pkcs12_enabled: "{{ true if fips_enabled|bool else false }}"
 
-### Boolean to enable cp-ansible's Custom Log4j Configuration across all components
+### Boolean to configure ZK, Kafka, Connect, and ksqlDBs logging with RollingFileAppender and log cleanup. Not necessary for other components
```

Comment:
Boolean to configure ZK, Kafka Broker's, Kafka Connect, and ksqlDB's logging with the RollingFileAppender and log cleanup functionality.

Reply:
Do you want the removal of "Not necessary for other components"? 

I am looking to say that the other components already have rollingfileappender+log cleanup.